packages/usa-character-count/src/index.js

```diff
@@ -150,9 +169,25 @@ const updateCountMessage = (inputEl) => {
     inputEl.setCustomValidity("");
   }
 
+  inputEl.classList.toggle(INPUT_ERROR_CLASS, isOverLimit);
   statusMessage.classList.toggle(MESSAGE_INVALID_CLASS, isOverLimit);
 };
 
+/**
+ * Updates character count form elements to match error status of input.
+ *
+ * @description On focusout, it will update the input label and form group to match the current state of the input element.
+ * Called on focusout instead of input to prevent content shift while typing.
+ * @param {HTMLInputElement|HTMLTextAreaElement} inputEl The character count input element
+ */
+const toggleErrorState = (inputEl) => {
+  const { labelEl, formGroupEl } = getCharacterCountElements(inputEl);
+  const errorState = inputEl.classList.contains(INPUT_ERROR_CLASS);
+
+  labelEl.classList.toggle(LABEL_ERROR_CLASS, errorState);
+  formGroupEl.classList.toggle(FORM_GROUP_ERROR_CLASS, errorState);
+};
+
```

Comment:
> [!NOTE]
> `toggleErrorState` adds/removes `form-group` and `usa-label` with relevant `--error` classes. Called on `focusout`

Reply:
Removed in [95a33e3](https://github.com/uswds/uswds/pull/5908/commits/95a33e323528e4c2310c487dc3fb9b3ea7b2e571)